diesel_tests/tests/schema.rs

```diff
@@ -58,11 +58,55 @@ pub struct Comment {
 }
 
 #[derive(PartialEq, Eq, Debug, Clone, Queryable)]
+#[has_many(special_comments)]
+#[belongs_to(user)]
+pub struct SpecialPost {
+    pub id: i32,
+    pub user_id: i32,
+    pub title: String
+}
+
+#[insertable_into(special_posts)]
+pub struct NewSpecialPost {
+    pub user_id: i32,
+    pub title: String
+}
+
+impl SpecialPost {
+    pub fn new(user_id: i32, title: &str) -> NewSpecialPost {
+        NewSpecialPost {
+            user_id: user_id,
+            title: title.to_owned()
+        }
+    }
+}
+
+#[derive(PartialEq, Eq, Debug, Clone, Queryable)]
+#[belongs_to(special_post)]
+pub struct SpecialComment {
+    pub id: i32,
+    pub special_post_id: i32,
+}
+
+impl SpecialComment {
+    pub fn new(special_post_id: i32) -> NewSpecialComment {
+        NewSpecialComment {
+            special_post_id: special_post_id
+        }
+    }
+}
+
+#[insertable_into(special_comments)]
+pub struct NewSpecialComment {
+    special_post_id: i32
+}
```

Comment:
Are we able to inline these structs into the test functions? This file is starting to become a dumping ground.